lib/chef/knife/vsphere_vm_find.rb

```diff
@@ -189,6 +193,10 @@ def run
        if get_config(:ip)
          print "#{ui.color("IP:", :cyan)} #{vmc.guest.ipAddress}\t"
        end
+       if get_config(:ips)
+         networks = vmc.guest.net.map { |net| net.network + ":" + net.ipAddress.select { |i| i[/^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$/] }[0] }.join(",")
```

Comment:
I was checking to see if this could possibly be a IPv6 address, and [the docs](https://www.vmware.com/support/developer/converter-sdk/conv55_apireference/vim.vm.GuestInfo.NicInfo.html) say that `guest.net.ipAddress` is deprecated in favour of `guest.net.ipConfig` which is a bit more complicated since it handles multiple addresses. What would you think about using `ipConfig` instead of `ipAddress`?